CHALLENGE.md

```diff
@@ -2,22 +2,23 @@
 
 ## Prerequisites
 
-- Clone this repository to your machine
-- Select the branch (depending on the language you want to work with)
-  - The `master` branch is based on TypeScript.
-
+- Clone this repository to your machine.
+- Please, avoid changing the repository structure without a strict necessity; otherwise, motivate your choice.
+- Write the code you are usually writing. Avoid writing broken or sketchy code. We can't evaluate how good you are if the code is not good.
+- In case of any questions regarding the challenge, please, contact a member of the hiring team who shared this challenge with you.
 ## Introduction
 
-You are given a prepared SPA project that was bootstrapped with Create React App. The app must render a list of tournaments from a fake REST API (included in the project). New tournaments can be created and tournaments should be searchable (search functionality must be optimised with performance in mind), editable and deletable through the fake REST API. A theme and a few basic elements such as headings, button and input have already been created for you.
+You are given a prepared SPA project that was bootstrapped with Create React App. The app must render a list of tournaments from a fake REST API (included in the project). New tournaments can be created and tournaments should be searchable, editable and deletable through the fake REST API. A theme and a few basic elements such as headings, button and input have already been created for you.
```

Comment:
This is a minimally structured Create React App SPA project.
The project includes:
- A theme.
- Basic components that are meant to be used by you. (If we don't mention the components here, we will see if the candidate is able to find them after we've told the candidate that there are some components for use).
- A fake REST API.